dappnode_package.json

```diff
@@ -30,7 +31,7 @@
     "url": "https://github.com/dappnode/DAppNodePackage-bee/issues"
   },
   "license": "GPL-3.0",
-  "mainService": "dashboard",
+  "mainService": "bee.dnp.dappnode.eth",
```

Comment:
Change service name of the node to `node`

Reply:
After doing that change(edit mainService to node and modify docker-compose service's name bee.dnp.dapnde.eth to node), there is a error after installing the package and set up the environment vars
```errorCall to packageSetEnvironment
Today, 5 min ago
Reply: No service bee.dnp.dappnode.eth in dnp bee.dnp.dappnode.eth
Error: No service bee.dnp.dappnode.eth in dnp bee.dnp.dappnode.eth
    at /usr/src/app/webpack:/src/calls/packageSetEnvironment.ts:28:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/index.js:66678:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

{
  "dnpName": "bee.dnp.dappnode.eth",
  "environmentByService": {
    "node": {
      "PASSWORD": "**********",
      "EXTRA_OPTS": "full-node"
    },
    "bee.dnp.dappnode.eth": {
      "PASSWORD": "**********",
      "BEE_SWAP_ENDPOINT": "http://goerli-geth.dappnode:8545",
      "BEE_DB_CAPACITY": "5000000",
      "BEE_WELCOME_MESSAGE": "Hello from my DAppNode!"
    }
  }
}
```